package.json

```diff
@@ -42,12 +42,10 @@
     "debug": "^4.1.1",
     "ember-try-config": "^3.0.0",
     "execa": "^1.0.0",
-    "extend": "^3.0.0",
     "fs-extra": "^5.0.0",
     "promise-map-series": "^0.3.0",
```

Comment:
Think the PR removes the last use of `promise-map-series`

Reply:
Ya, I think so too, I missed that one when I was removing the others.